test/integration/restart_task_test.rb

```diff
@@ -0,0 +1,108 @@
+# frozen_string_literal: true
+require 'test_helper'
+require 'kubernetes-deploy/restart_task'
+
+class RestartTaskTest < KubernetesDeploy::IntegrationTest
+  def test_restart
+    deploy_fixtures("basic", subset: ["configmap-data.yml", "web.yml.erb"])
+
+    deployment = fetch_deployment("web")
+    refute fetch_restarted_at(deployment), "no RESTARTED_AT env on fresh deployment"
```

Comment:
You have these two lines in a few places here `assert_fetched_at deployment("web")` might be convenient and a little more readable 👀